config/printer-creality-ender3-s1-2021.cfg

```diff
@@ -0,0 +1,109 @@
+#Compile Firmware with:
+#STM32 => STM32F103
+#Bootloader offset 28KiB
+#Clock Reference 8MHz
+#Communication interface USART1 PA10/PA9 for using built-in USB-Port
+# See docs/Config_Reference.md for a description of parameters.
+
+[stepper_x]
+step_pin: PC2
+dir_pin: PB9
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 40
+endstop_pin: ^!PA5
+position_endstop: 0
+position_max: 220
+homing_speed: 50
+
+[stepper_y]
+step_pin: PB8
+dir_pin: PB7
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 40
+endstop_pin: ^!PA6
+position_endstop: 0
+position_max: 220
+homing_speed: 50
+
+[stepper_z]
+step_pin: PB6
+dir_pin: !PB5
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 8
+endstop_pin: probe:z_virtual_endstop
+position_max: 270
+
+[extruder]
+step_pin: PB4
+dir_pin: PB3
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 2.13667236
+gear_ratio: 1:3.5
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: PA1
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: PC5
+control: pid
+pid_Kp: 26.949
+pid_Ki: 1.497
+pid_Kd: 121.269
+min_temp: 0
+max_temp: 250
+
+[heater_bed]
+heater_pin: PA7
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: PC4
+control: pid
+pid_Kp: 327.11
+pid_Ki: 19.20
+pid_Kd: 1393.45
+min_temp: 0
+max_temp: 130
+
+[heater_fan nozzle_cooling_fan]
+pin: PC0
+
+[fan]
+pin: PA0
+
+[mcu]
+serial: /dev/serial/by-id/usb-1a86_USB_Serial-if00-port0
+restart_method: command
+
+[printer]
+kinematics: cartesian
+max_velocity: 300
+max_accel: 3000
+max_z_velocity: 5
+max_z_accel: 100
+
+[bltouch]
+sensor_pin: ^PC14
+control_pin: PC13
+x_offset: -34.2
+y_offset: -40.7
+z_offset: 0
+probe_with_touch_mode: true
+stow_on_each_sample: false
+
+[bed_mesh]
+speed: 120
+mesh_min: 15, 15
+mesh_max: 200,200
+probe_count: 4,4
+algorithm: bicubic
+
+[safe_z_home]
+home_xy_position:110,110
+speed: 75
+z_hop: 5
+z_hop_speed: 5
+
+[filament_switch_sensor e0_sensor]
```

Comment:
Does this come with a filament sensor?  The product page says it doesn't, but it kind of looks like there is one in the photograph.